src/windows/Accessibility.ts

```diff
@@ -3,17 +3,32 @@
 *
 * Copyright (c) Microsoft Corporation. All rights reserved.
 * Licensed under the MIT license.
-*
-* Web wrapper for subscribing or querying the current state of the
-* screen reader.
 */
 
-import { Accessibility as CommonAccessibility } from '../common/Accessibility';
+import RN = require('react-native');
+import { Accessibility as NativeAccessibility } from '../native-common/Accessibility';
+import SyncTasks = require('synctasks');
+
+export class Accessibility extends NativeAccessibility {
+    private _isHighContrast = RN.AccessibilityInfo.initialHighContrast;
+
+    constructor() {
+        super();
```

Comment:
>super(); [](start = 8, length = 8)

We should call `fetchIsHighContrast` to be sure that `initialHighContrast` isn't stale.